src/components/MapContainer.svelte

```diff
@@ -1,50 +1,31 @@
+<!-- @migration-task Error while migrating Svelte code: $$props is used together with named props in a way that cannot be automatically migrated. -->
```

Comment:
looks like there's some more work to do here.

Reply:
No, I forgot to remove the comment. It appeared because the migration script failed to remove $$props, but it has been migrated now to Svelte 5 so there is no errors .